yb-voyager/cmd/getDataMigrationReportCommand.go

```diff
@@ -184,7 +183,38 @@ func addRowInTheTable(uitbl *uitable.Table, row rowData) {
 	uitbl.AddRow(row.TableName, row.DBType, row.ExportedSnapshotRows, row.ImportedSnapshotRows, row.ExportedInserts, row.ExportedUpdates, row.ExportedDeletes, row.ImportedInserts, row.ImportedUpdates, row.ImportedDeletes, getFinalRowCount(row))
 }
 
-func updateImportedEventsCountsInTheRow(row *rowData, tableName string, schemaName string, targetConf *tgtdb.TargetConf) error {
+func updateExportedSnapshotRowsInTheRow(msr *metadb.MigrationStatusRecord, row *rowData, tableName string, schemaName string, dbzmStatus *dbzm.ExportStatus, snapshotDataFileDescriptor *datafile.Descriptor) {
+	// TODO: read only from one place(data file descriptor). Right now, data file descriptor does not store schema names.
+	if msr.SnapshotMechanism == "debezium" {
+		tableExportStatus := dbzmStatus.GetTableExportStatus(tableName, schemaName)
+		if tableExportStatus == nil {
+			tableExportStatus = &dbzm.TableExportStatus{
+				TableName:                tableName,
+				SchemaName:               schemaName,
+				ExportedRowCountSnapshot: 0,
+				FileName:                 "",
+			}
+		}
+		row.ExportedSnapshotRows = tableExportStatus.ExportedRowCountSnapshot
+	} else {
+		if msr.SourceDBConf.DBType == "postgres" && schemaName != "public" && schemaName != "" {
```

Comment:
DBType for pg is "postgresql", or you can use the const `POSTGRESQL`

Reply:
oops, forgot to test this case. THanks, fixed!